### ISSUE TITLE
Consistent style across the site

### ISSUE THREAD

**Issue by yuchenliu15** [BUGS-NYU/bugs-nyu.github.io-old#218]
Need to update styles of other pages (team, alumni, contributors) to stay consistent with the new home page style

![image](https://user-images.githubusercontent.com/43976449/116339722-cc533100-a792-11eb-87ef-61e254aaafe3.png)

Home page:
![image](https://user-images.githubusercontent.com/43976449/116339875-12a89000-a793-11eb-997c-69a525a45cd0.png)

**Comment by Kn99HN** [comment]
Yep, ideally, I'd like an UI/UX designer to provide a Figma design so we can implement it easily. It's a bit hard to go without design since the page has grown a bit larger than expected
